lib/src/widget/button/app_ghost_button.dart

```diff
@@ -93,15 +103,42 @@ class AppGhostButton extends StatelessWidget {
     );
     return Opacity(
       opacity: onPressed == null ? linden.dimen.disabledOpacity : 1,
-      child: TextButton(
-        style: style,
-        onPressed: onPressed,
-        onLongPress: onLongPressed,
-        child: _buildChild(linden),
-      ),
+      child: _buildTextButton(linden: linden, style: style),
     );
   }
 
+  Widget _buildTextButton({
+    required Linden linden,
+    required ButtonStyle style,
+  }) {
+    final textButton = TextButton(
+      style: style,
+      onPressed: onPressed,
+      onLongPress: onLongPressed,
+      child: _buildChild(linden),
+    );
+
+    /// If the button contains only an icon
+    /// set the size to 48x48
+    if (isWithIconOnly) {
+      return SizedBox(
+        height: linden.dimen.unit6,
+        width: linden.dimen.unit6,
```

Comment:
🛑 there in the dimen should be `buttonIconSize` or so, please use them

Reply:
Done!! :)